src/main/scala/wdltool/GraphPrint.scala

```diff
@@ -10,15 +10,12 @@ import scala.collection.JavaConverters._
 
 import cats._
 import cats.implicits._
+import cats.derived.monoid._, legacy._
```

Comment:
Purely OOI: I can guess why the `monoid._` is needed, but do you know what `legacy._` is giving us?